### PR TITLE
[Support new order funnel - Part 6] Refactor legacy dashboard

### PR DESCRIPTION
**Pull request by FredericEspiau**
# Part 6 - Allow upgrading resources for all subsidiaries
This PR is part of the **Support new order funnel** project

: construction: this is a work still in progress

## 💅 Refactoring
* The dashboard for legacy PCC products has been refactored
    * It should almost work the same, the only difference is that changing name displays an information message

## :rocket: Features
* Options and Certifications dashboard tile :
    * Add options to order / upgrade service pack and certifications
(This part was in first place meant to be made through another PR, but the work is now made with this one)

## References
- MANAGER-2409

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci#1229.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_3#1229.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_2#1229.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report build-manager#1235.0 ■   
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ■   
  * Build manager EU ■   
  * Build manager US ■   
  * Tests ✔

File: client/app/dedicatedCloud/dashboard/main/tiles/options/order/steps/selection/style.less [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    top: 3rem;
+    left: 2.5rem;
+  }
+}
+
+.dedicatedCloud-servicePack-option-section {
+  margin-left: 2rem;
+  margin-right: 2rem;
+  width: initial;
+}
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply client/app/dedicatedCloud/dashboard/main/tiles/options/order/steps/selection/style.less]: Hmm tested with `ml-x mr-x` or `mx-x`, it doesn't work

File: client/app/dedicatedCloud/dashboard/main/tiles/options/order/component.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    $transition$: '<',
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply client/app/dedicatedCloud/dashboard/main/tiles/options/order/component.js]: Look at https://github.com/ovh-ux/ovh-manager-dedicated/pull/1031/files/e09591e7d90d464e9c6fda4666cbe5e03aaf9139..fc04a0fce55167a4e80cbe0885cb42f231794856#diff-91928623c06351d0ae32db2df6337eebR20 ;)

File: client/app/dedicatedCloud/dashboard/legacy/controller.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    this.allowDedicatedServerComplianceOptions = this.constants.target !== 'US';
+
+    this.setAction = (action, data) => this.$scope.$parent.setAction(action, data);
+  }
+
+  openModalToEditDescription() {
+    return this.$uibModal
+      .open({
+        animation: true,
+        templateUrl: 'components/name-edition/name-edition.html',
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply client/app/dedicatedCloud/dashboard/legacy/controller.js]: Hmm I'm ok with your idea, but the fact is importing the template would result in a `../../../longpathToTemplate`.
Also making the modal a `state` will cancel the `then` callback

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**